Makefile

```diff
@@ -595,6 +612,30 @@ $(BUILD_DIR)/assets/%.jpg.inc.c: assets/%.jpg
 $(BUILD_DIR)/assets/%.jpg.inc.c: $(EXTRACTED_DIR)/assets/%.jpg
 	$(ZAPD) bren -eh -i $< -o $@
 
+# Audio
+
+AUDIO_BUILD_DEBUG ?= 0
+
+# first build samples...
+
+$(BUILD_DIR)/assets/audio/samples/%.half.aifc: assets/audio/samples/%.half.wav
+	$(SAMPLECONV) vadpcm-half $< $@
+
+$(BUILD_DIR)/assets/audio/samples/%.half.aifc: $(EXTRACTED_DIR)/assets/audio/samples/%.half.wav
+	$(SAMPLECONV) vadpcm-half $< $@
+ifeq ($(AUDIO_BUILD_DEBUG),1)
+	@(cmp $(<D)/aifc/$(<F:.half.wav=.half.aifc) $@ && echo "$(<F) OK") || (mkdir -p NONMATCHINGS/$(<D) && cp $(<D)/aifc/$(<F:.half.wav=.half.aifc) NONMATCHINGS/$(<D)/$(<F:.half.wav=.half.aifc))
+endif
```

Comment:
Could you add a TODO to remove these also?

Reply:
I'm not sure what the timeline would be for removing these, I wouldn't be that comfortable removing them until every version is confirmed matching. I can add a TODO up near where `AUDIO_BUILD_DEBUG` is defined, but I suspect there's lots of things that would be removed by the time I'd want to remove this, that aren't covered by a TODO.

Reply:
once the tables are build from code, would first_diff.py show the name of a broken sample?

Reply:
I think first_diff would be able to say which sample is non-matching since every sample would have a symbol. However, just knowing that isn't enough, when debugging I'd dump the binary to compare with the original in isolation, which is all this does (but automatically)

Reply:
does this actually do any dumping? it seems like it only moves existing files around

Reply:
moving and dumping are equivalent here since there are no pointers or other data modified by linking